### ISSUE TITLE
Wrong mention span initialization from gold corefs

### ISSUE THREAD

**Issue by rupimanoj** [huggingface/neuralcoref#160]
In the conllparser.py script, when code is configured to use gold corefs I believe there is a mistake in creating mentions for a document.
```
if use_gold_mentions:
            for coref in corefs:
                #            print("coref['label']", coref['label'])
                #            print("coref text",parsed[coref['start']:coref['end']])
                mention = Mention(parsed[coref['start']:coref['end']], len(self.mentions), len(self.utterances),
                                  self.n_sents, speaker=self.speakers[speaker_id], gold_label=coref['label'])
                self.mentions.append(mention)
                #            print("mention: ", mention, "label", mention.gold_label)
```
In the above code, for the first argument in the Mention constructor arguments, it should be ```parsed[coref['start']:coref['end']+1] ```. Please correct me if I am wrong.

**Comment by svlandeg** [comment]
@rupimanoj : thanks for the report!

Sorry for the late response. I haven't checked myself yet, but this fixes an issue discussed in #208 :-)

**Comment by svlandeg** [comment]
Issue addressed by PR #213 - thanks @cbiehl and @rupimanoj !
